src/plugins/displayLayout/components/DisplayLayout.vue

```diff
@@ -254,7 +294,26 @@
                 }
             },
             removeChild(identifier) {
-                // TODO: implement
+                let keyString = this.openmct.objects.makeKeyString(identifier);
+
+                if (this.objectViewMap[keyString]) {
+                    delete this.objectViewMap[keyString];
+                    this.removeItemFromConfiguration(keyString);
```

Comment:
can move this out of if/else block so you don't have to repeat it in both places.

Reply:
It should only be called when objectViewMap or telemetryViewMap has the keyString, so it can't be moved out of if/else.